contracts/samples/SimpleAccount.sol

```diff
@@ -104,8 +104,9 @@ contract SimpleAccount is BaseAccount, TokenCallbackHandler, UUPSUpgradeable, In
     /// implement template method of BaseAccount
     function _validateSignature(PackedUserOperation calldata userOp, bytes32 userOpHash)
     internal override virtual returns (uint256 validationData) {
-        bytes32 hash = MessageHashUtils.toEthSignedMessageHash(userOpHash);
-        if (owner != ECDSA.recover(hash, userOp.signature))
+
+        //userOpHash can be generated using eth_signTypedData_v4
```

Comment:
This comment is unnecessary

Reply:
I think it is important, as there is no other place in our codebase that explains that this sig is directly a signed-typed-data signature (note that the old signature that was removed by this commit explicitly had "toEthSignedMessageHash", but for the new signature, you need to look back to where we calculate the userOpHash to see it is an erc-712 signature